projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk-jupyterlab-extension/vdk.py

```diff
@@ -56,3 +57,20 @@ def delete_job(name: str, team: str, rest_api_url: str):
             return f"Deleted the job with name {name} from {team} team. "
         except Exception as e:
             raise e
+
+    @staticmethod
+    def download_job(name: str, team: str, rest_api_url: str, path: str):
+        """
+        Execute `download job`.
+        :param name: the name of the data job that will be downloaded
+        :param team: the team of the data job that will be downloaded
+        :param rest_api_url: The base REST API URL
+        :param path: the path to the directory where the job will be downloaded
+        :return: message that the job is downloaded
+        """
+        try:
+            cmd = JobDownloadSource(rest_api_url)
+            cmd.download(team, name, path)
+            return f"Downloaded the job with name {name} to {path}. "
+        except Exception as e:
```

Comment:
why catch it just to re raise it? 
Can you just not catch it at all and and let the calling code handle it.